source/sink-connector/fundamentals/post-processors.txt

```diff
@@ -187,6 +187,17 @@ sink connector.
        | For information on configuration, see the
          :ref:`Renaming by Regular Expression Example <field-renamer-regex-example>`.
 
+   * - | **NullFieldValueRemover**
+     - | **Full Path:**
+
+       .. code-block:: none
+
+          com.mongodb.kafka.connect.sink.processor.NullFieldValueRemover``
+
+       | Removes all null fields from the sink record.
```

Comment:
Can we reword this to say "Removes all document fields that contain null values from the sink record."